src/main/java/com/raoulvdberge/refinedstorage/item/CapabilityProviderEnergy.java

```diff
@@ -11,12 +11,10 @@
 import javax.annotation.Nullable;
 
 public class CapabilityProviderEnergy implements ICapabilityProvider {
```

Comment:
The changes in this file are dangerous, and involve capability creation semantics. Please undo.